cmd/libs/go2idl/client-gen/generators/client_generator.go

```diff
@@ -39,13 +39,15 @@ func NameSystems() namer.NameSystems {
 	pluralExceptions := map[string]string{
 		"Endpoints": "Endpoints",
 	}
+	lowercaseNamer := namer.NewAllLowercasePluralNamer(pluralExceptions)
 	return namer.NameSystems{
 		"public":             namer.NewPublicNamer(0),
 		"private":            namer.NewPrivateNamer(0),
 		"raw":                namer.NewRawNamer("", nil),
 		"publicPlural":       namer.NewPublicPluralNamer(pluralExceptions),
 		"privatePlural":      namer.NewPrivatePluralNamer(pluralExceptions),
-		"allLowercasePlural": namer.NewAllLowercasePluralNamer(pluralExceptions),
+		"allLowercasePlural": lowercaseNamer,
+		"resource":           NewTagOverrideNamer("resourceName", lowercaseNamer),
```

Comment:
What do you mean?  `k8s.io/metrics` already makes use of the `resourceName` tag...